test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -1666,8 +1666,39 @@ def check_number_of_cooling_towers(prototypes_base)
     end
   end
 
-  # Set ZoneMultiplier to passed value for all zones
+  # Check if the VAV box minimum flow setpoint are
+  # assigned following the rules in Appendix G
+  #
+  # @param prototypes_base [Hash] Baseline prototypes
+  def check_vav_min_sp(prototypes_base)
+    standard = Standard.build('90.1-PRM-2019')
+    prototypes_base.each do |prototype, model|
+      building_type, template, climate_zone, mod = prototype
 
+      model.getAirLoopHVACs.each do |air_loop|
+        air_loop.thermalZones.each do |zone|
+          zone.equipment.each do |equip|
+            if equip.to_AirTerminalSingleDuctVAVReheat.is_initialized
+              zone_oa = standard.thermal_zone_outdoor_airflow_rate(zone)
+              vav_terminal = equip.to_AirTerminalSingleDuctVAVReheat.get
+              expected_mdp = [zone_oa / vav_terminal.autosizedMaximumAirFlowRate.get, 0.3].max.round(2)
+              actual_mdp = vav_terminal.constantMinimumAirFlowFraction.get.round(2)
+              assert(expected_mdp == actual_mdp , "Minimum MDP for #{building_type} for #{template} in #{climate_zone} should be #{expected_mdp} but #{actual_mdp} is used in the model.")
+            elsif equip.to_AirTerminalSingleDuctParallelPIUReheat.is_initialized
+              zone_oa = standard.thermal_zone_outdoor_airflow_rate(zone)
+              fp_vav_terminal = equip.to_AirTerminalSingleDuctParallelPIUReheat.get
+              expected_prim_frac = [zone_oa / fp_vav_terminal.autosizedMaximumPrimaryAirFlowRate.get, 0.3].max.round(2)
+              actual_prim_frac = fp_vav_terminal.minimumPrimaryAirFlowFraction.get
+              assert(expected_prim_frac == actual_prim_frac , "Minimum primary air flow fraction for #{building_type} for #{template} in #{climate_zone} should be #{expected_prim_frac} but #{actual_prim_frac} is used in the model.")
+            end
+          end
+        end
+      end
+    end  
+  end
+
+  # Set ZoneMultiplier to passed value for all zones
+  #
```

Comment:
New test to check the MDP/Primary air flow fraction of two of the prototypes baseline models: medium office (FPB) and primary school (standard VAV boxes).